tests/ImageSharp.Tests/Formats/Png/PngDecoderTests.cs

```diff
@@ -242,10 +242,10 @@ public void DetectPixelSize(string imagePath, int expectedPixelSize)
         }
 
         [Theory]
-        [InlineData(PngChunkTypes.Header)]
-        [InlineData(PngChunkTypes.Palette)]
+        [InlineData("IHDR")] // Header
```

Comment:
Could we refactor this test to use the `PngChunkType` enum + `BinaryPrimitives`?